app/javascript/components/upload/AnnDataStep.jsx

```diff
@@ -4,11 +4,12 @@ import AnnDataFileForm from './AnnDataFileForm'
 import { AddFileButton } from './form-components'
 
 const DEFAULT_NEW_ANNDATA_FILE = {
-  file_type: 'AnnData',
+  file_type: 'AnnData Reference',
   options: {}
 }
 
-const AnnDataFileFilter = file => ['AnnData'].includes(file.file_type)
+const AnnDataFileTypes = ['AnnData Reference', 'AnnData Ingestible']
```

Comment:
Blocker: for succinctness and consistency, let's retain "AnnData" and interpret the lack of qualifiers as "AnnData ingestible".  It seems more maintainable in the long term to consider the default "AnnData" type to be ingestible, rather than needing to pass around the explicit qualifier.

This convention would let us simplify naming for other foreseeable future use cases, like reference and ingestible Seurat files.

```suggestion
const AnnDataFileTypes = ['AnnData', 'AnnData Reference']
```